db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20010101010101) do
+ActiveRecord::Schema.define(:version => 20150901132832) do
```

Comment:
But that shouldn't be - I've run a lot of migrations, and I haven't seen that happen. The schema table keeps a list of which ones have run.